steveDevonport/steve_devonport.md

```diff
@@ -0,0 +1,46 @@
+
+When is a hero not a hero?
+
+In a previous role I was tasked with creating the Test Automation for a number of different web services, however when I joined the team I quickly discovered that they were lacking the capability to deliver customer value rapidly, securely, or with predictability. All this was compounded by the Scrum ceremonies not being orchestrated and there was no Scrum Master. The technology and tooling side was no better.
+
+After being convinced by the recruitment team and interview process that I would be working with an experienced and high performant group of people the above scenario came as a complete shock, whilst there might be a high performing group of people somewhere in the company...I guess I drew the short straw.
+
+As I saw it I had 2 choices...(1) quit and find a new role or (2) stick around and try to turn the team around.
+
+It's fair to say I chose option 2 otherwise you wouldn't be reading all about my experiences of working with a hero.
+
+As I got to know the people on the team, over the first several months, I put together a list of 'things' that were either missing or just seemed wrong. And so it was that before the end of my probation period I presented my manager with a three pronged approach that would allow us to deliver features transparently, at pace, and with reliability - all within 12 months. A critical consideration to my approach was (1) My Product Owner fully supported the approach and (2) For the first 6 months the team would deliver minimal features into production.
```

Comment:
Were the team doing anything right?  It would be good to emphasis the good as well as the bad.

Reply:
also, how did you get 'buy in' for your approach - it would be good to explain that to the readers.